recipe/meta.yaml

```diff
@@ -22,23 +21,25 @@ requirements:
     - numpy >=1.21.2                         # [build_platform != target_platform]
     - pybind11                               # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```

Comment:
This seems to have broken the build. Do you think it can be removed?

Reply:
According to [this](https://github.com/conda-forge/conda-forge.github.io/issues/2102#issuecomment-2225802928), it looks like I just need to re-render, which I will do.

Reply:
Great, thanks a lot for the effort! I'll merge the PR.